quickwit-search/src/leaf.rs

```diff
@@ -121,23 +125,83 @@ pub(crate) async fn open_index(
 ///
 /// The downloaded data depends on the query (which term's posting list is required,
 /// are position required too), and the collector.
+///
+/// * `query` - query is used to extract the terms and the fields which will be loaded from the
```

Comment:
in which case do we load a field due to the query? what does  a field mean here? Is it a fast field?

Reply:
I just commented on the function to document the behaviour. field is a tantivy `Field` here.